src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/brgemm_cpu_blocking.hpp

```diff
@@ -20,6 +20,22 @@ class BrgemmCPUBlocking : public ov::snippets::lowered::pass::BrgemmBlocking<Brg
 public:
     OPENVINO_RTTI("BrgemmCPUBlocking", "BrgemmBlocking")
 
+    /**
+     * @interface DummyPass
+     * @brief The empty pass which is used to force insertion of first specific iteration of loop by K dimension
+     *        Note: the pass is in public section to have opportunity to validate blocking loop in tests
+     * @ingroup snippets
+     */
+    class DummyPass : public snippets::lowered::pass::RangedPass {
+    public:
+        DummyPass() = default;
+        OPENVINO_RTTI("DummyPass", "RangedPass")
+        bool run(snippets::lowered::LinearIR& linear_ir,
+                 snippets::lowered::LinearIR::constExprIt begin,
+                 snippets::lowered::LinearIR::constExprIt end) override;
+        std::shared_ptr<snippets::lowered::pass::PassBase> merge(const std::shared_ptr<snippets::lowered::pass::PassBase>& other) override;
+    };
+
```

Comment:
The main remaining concern is that we have to keep `DummyPass` in cpu blocking pass (and `SetBrgemmBeta` in TPP one) in public section just because we need these passes to build the reference LIRs in tests. Ideally, these iterations handlers should be in private section.

However, there is no obvious solution for that so we can probably return to that discussion later. I don't think this blocks the merge